### PR TITLE
Modify Component

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -95,7 +95,7 @@ \subsubsection{Broker}
...
+\textbf{IOTA}\cite{IOTAwhitepaper} is a cryptocurrency for the IoT industry based on a revolutionary distributed ledger technology, the \textbf{Tangle}, which the ledger consists of a directed acyclic graph(DAG) of individual transactions. To issue a transaction, one needs to perform a small amount of Proof-of-Work (PoW) that verifies two previous transactions. Since every actor in Tangle has the same role, IOTA is feeless with high scalability . Moreover, it is possible to make \textbf{zero-transactions} that one could store information securely within Tangle transactions.
```
Comment (jserv) [inline paper.tex]: I would like to mention MAM here. IOTA should be referred within the paragraph about MAM. The concept of second layer data communication protocol such as MAM is important for our research. However, IOTA is not vital from the perspective of Interchangeability. Thus, let's merge the descriptions about MAM and IOTA into single component part.

File: paper.tex [hunk 2]
```diff
@@ -104,16 +104,17 @@ \subsubsection{IPFS}
...
 
-\subsubsection{TangleID}
-TangleID\cite{TangleID} is a self-soverign identity system based on IOTA, the goal is to establish a solution that do not require any third-party authority to verify an identity and its digital footprint. In TangleID, the digital footprint is converted into digital assets with the principle of Decentralized Identifiers (DIDs)\cite{DID} defined by W3C, and by putting DID documents on MAM makes TangleID a GDPR-Complianced system. Each identity has a public/private key pair recorded on MAM, it is used in digital signature for identity authentication. With every participant in data marketplace registers on TangleID, one can easily verify data provider's identity and data integrity with the public key on the DID document, and that we can ensure the reliability of data sources.
+However, MAM resolves this problem, it publishes each message to different addresses that each can be derived from the previous one, this forms a MAM channel. Furthermore, an session key can be provided to control the visibility and access to a MAM channel.
```
Comment (twzjwang) [inline paper.tex]: "an session key" should be "a session key"

File: paper.tex (hunk 1 above)
Comment (jkrvivian) [reply paper.tex]: How about the order of each component? Do we need to reorder them?

Comment (jkrvivian) [reply paper.tex]: I'll reorder to:
* MAM
* TangleID
* Ethereum smart contract
* blind signature
* IPFS

Comment (jserv) [reply paper.tex]: The proposed order looks good to me.

**Comment by jserv** [comment]
Fix the conflict.

**Pull request merged**